meetups/1910/plan.md

```diff
@@ -0,0 +1,28 @@
+# October 2019 - Meetup plan
+
+## Lead
+
+Special one: collaborative organization / no pre-defined lead
+
+## Agenda
+
+* Hello world (2.0)
+  * Presenting ourselves
+  * Presenting resources
+  * Presenting plans
+
+* Presentation slot 1
+
+* Presentation slot 2
+
+* Dinner in the venue
+  * for future ones, drinks outside the venue is a good plan,
+  but we want to maximize audience for this first one
+
+## Sponsor
+
+TBD
+
+## Location
+
+TBD
```

Comment:
Adevinta could host it...

Reply:
Great! I will add that. Depending also where the speakers come from, it may have also sense to accept venue offers from them too... but yeah, lets keep that as a potential plan